packages/web/src/index.tsx

```diff
@@ -2,11 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
-import { store } from './store';
+import { createApplicationStore } from './store';
 import { App } from './components/App';
 import { initAnalytics } from './services/analytics';
 import 'styles/global.scss';
 
+declare global {
+  interface Window {
+    __INITIAL_STATE__?: object;
+  }
+}
```

Comment:
I think global.d.ts in web package root is a better place for this

Reply:
```
TS2669: Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.
```

Reply:
You just don't need the `declare global` wrapper there.